.github/workflows/release.yml

```diff
@@ -0,0 +1,43 @@
+name: Release SPRAS on new tags
+# This action runs whenever a new tag matching vX.Y.Z is pushed to the repo.
```

Comment:
Adjust depending on what we decide below regarding tag pattern

Reply:
Fixed in the latest commit.